test/runtests.jl

```diff
@@ -0,0 +1,17 @@
+using ChainRulesCore
+using ChainRulesTestUtils
+using Random
+using Test
+
+@testset "ChainRulesTestUtils.jl" begin
+    double(x) = 2x
+    @scalar_rule(double(x), 2)
+    test_scalar(double, 2)
+
+    fst(x, y) = x
+    ChainRulesCore.frule(::typeof(fst), x, y, _, dx, dy) = (x, dx)
+    ChainRulesCore.rrule(::typeof(fst), x, y) = fst_pullback(dout) = (NO_FIELDS, @thunk(identity(dout)), Zero())
```

Comment:
```suggestion
    function ChainRulesCore.rrule(::typeof(fst), x, y)
        function fst_pullback(Δx)
            return (NO_FIELDS, Δx, Zero())
        end
        return x, fst_pullback
    end
```
An `rrule` need to return both the forwards pass (`fst(x, y)`) and the pullback.